var/spack/repos/builtin/packages/py-pennylane-lightning/package.py

```diff
@@ -47,8 +48,8 @@ class PyPennylaneLightning(CMakePackage, PythonExtension):
 
     # variant defined dependencies
     depends_on("blas", when="+blas")
-    depends_on("kokkos@3.7.00", when="+kokkos")
-    depends_on("kokkos-kernels@3.7.00", when="+kokkos")
+    depends_on("kokkos@:3.7.01", when="+kokkos")
+    depends_on("kokkos-kernels@:3.7.01", when="+kokkos")
```

Comment:
Should we update it?
```suggestion
    depends_on("kokkos@:4.0.01", when="+kokkos")
    depends_on("kokkos-kernels@:4.0.01", when="+kokkos")
```

Reply:
I believe we are at this moment depending on different Kokko's versions in `lightning` and `lightning-kokkos`.

Reply:
This is only for the released packages, so everything should be hard pinned to 3.7
We can move to 4.0 for the 0.31 release.